### ISSUE TITLE
Issue reporter appears without reason

### ISSUE THREAD

**Issue by trierweiler** [spyder-ide/spyder#15825]
### What steps will reproduce the problem?

I've just created a new conda environment and conda install spyder.

After opening spyder I closed a project that was already loaded and created a new one with a custom location.

That's all!

### Traceback
```python-traceback
--- Logging error ---
Traceback (most recent call last):
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 113, in get_comm
    return self.comms[comm_id]
KeyError: 'a24ab403c93811eb96ee5cc9d3a28a56'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\logging\__init__.py", line 1086, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "C:\Users\eng_g\Anaconda3\envs\workdb\Scripts\spyder-script.py", line 10, in <module>
    sys.exit(main())
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\spyder\app\start.py", line 228, in main
    mainwindow.main(options, args)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\spyder\app\mainwindow.py", line 2293, in main
    mainwindow = create_window(app, splash, options, args)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\spyder\app\mainwindow.py", line 2192, in create_window
    app.exec_()
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 48, in _dispatch
    getattr(self, msg_type)(msg)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 172, in comm_msg
    comm = self.get_comm(comm_id)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 117, in get_comm
    self.log.warning("No such comm: %s", comm_id)
Message: 'No such comm: %s'
Arguments: ('a24ab403c93811eb96ee5cc9d3a28a56',)
--- Logging error ---
Traceback (most recent call last):
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 113, in get_comm
    return self.comms[comm_id]
KeyError: 'a24ab403c93811eb96ee5cc9d3a28a56'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\logging\__init__.py", line 1086, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "C:\Users\eng_g\Anaconda3\envs\workdb\Scripts\spyder-script.py", line 10, in <module>
    sys.exit(main())
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\spyder\app\start.py", line 228, in main
    mainwindow.main(options, args)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\spyder\app\mainwindow.py", line 2293, in main
    mainwindow = create_window(app, splash, options, args)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\spyder\app\mainwindow.py", line 2192, in create_window
    app.exec_()
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 48, in _dispatch
    getattr(self, msg_type)(msg)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 172, in comm_msg
    comm = self.get_comm(comm_id)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 117, in get_comm
    self.log.warning("No such comm: %s", comm_id)
Message: 'No such comm: %s'
Arguments: ('a24ab403c93811eb96ee5cc9d3a28a56',)
--- Logging error ---
Traceback (most recent call last):
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 113, in get_comm
    return self.comms[comm_id]
KeyError: 'a24ab403c93811eb96ee5cc9d3a28a56'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\logging\__init__.py", line 1086, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "C:\Users\eng_g\Anaconda3\envs\workdb\Scripts\spyder-script.py", line 10, in <module>
    sys.exit(main())
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\spyder\app\start.py", line 228, in main
    mainwindow.main(options, args)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\spyder\app\mainwindow.py", line 2293, in main
    mainwindow = create_window(app, splash, options, args)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\spyder\app\mainwindow.py", line 2192, in create_window
    app.exec_()
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 48, in _dispatch
    getattr(self, msg_type)(msg)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 172, in comm_msg
    comm = self.get_comm(comm_id)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 117, in get_comm
    self.log.warning("No such comm: %s", comm_id)
Message: 'No such comm: %s'
Arguments: ('a24ab403c93811eb96ee5cc9d3a28a56',)
--- Logging error ---
Traceback (most recent call last):
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 113, in get_comm
    return self.comms[comm_id]
KeyError: 'a24ab403c93811eb96ee5cc9d3a28a56'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\logging\__init__.py", line 1086, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "C:\Users\eng_g\Anaconda3\envs\workdb\Scripts\spyder-script.py", line 10, in <module>
    sys.exit(main())
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\spyder\app\start.py", line 228, in main
    mainwindow.main(options, args)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\spyder\app\mainwindow.py", line 2293, in main
    mainwindow = create_window(app, splash, options, args)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\spyder\app\mainwindow.py", line 2192, in create_window
    app.exec_()
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 48, in _dispatch
    getattr(self, msg_type)(msg)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 172, in comm_msg
    comm = self.get_comm(comm_id)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 117, in get_comm
    self.log.warning("No such comm: %s", comm_id)
Message: 'No such comm: %s'
Arguments: ('a24ab403c93811eb96ee5cc9d3a28a56',)
--- Logging error ---
Traceback (most recent call last):
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 113, in get_comm
    return self.comms[comm_id]
KeyError: 'a24ab403c93811eb96ee5cc9d3a28a56'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\logging\__init__.py", line 1086, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "C:\Users\eng_g\Anaconda3\envs\workdb\Scripts\spyder-script.py", line 10, in <module>
    sys.exit(main())
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\spyder\app\start.py", line 228, in main
    mainwindow.main(options, args)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\spyder\app\mainwindow.py", line 2293, in main
    mainwindow = create_window(app, splash, options, args)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\spyder\app\mainwindow.py", line 2192, in create_window
    app.exec_()
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 48, in _dispatch
    getattr(self, msg_type)(msg)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 172, in comm_msg
    comm = self.get_comm(comm_id)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 117, in get_comm
    self.log.warning("No such comm: %s", comm_id)
Message: 'No such comm: %s'
Arguments: ('a24ab403c93811eb96ee5cc9d3a28a56',)
--- Logging error ---
Traceback (most recent call last):
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 113, in get_comm
    return self.comms[comm_id]
KeyError: 'a24ab403c93811eb96ee5cc9d3a28a56'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\logging\__init__.py", line 1086, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "C:\Users\eng_g\Anaconda3\envs\workdb\Scripts\spyder-script.py", line 10, in <module>
    sys.exit(main())
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\spyder\app\start.py", line 228, in main
    mainwindow.main(options, args)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\spyder\app\mainwindow.py", line 2293, in main
    mainwindow = create_window(app, splash, options, args)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\spyder\app\mainwindow.py", line 2192, in create_window
    app.exec_()
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 48, in _dispatch
    getattr(self, msg_type)(msg)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 172, in comm_msg
    comm = self.get_comm(comm_id)
  File "C:\Users\eng_g\Anaconda3\envs\workdb\lib\site-packages\qtconsole\comms.py", line 117, in get_comm
    self.log.warning("No such comm: %s", comm_id)
Message: 'No such comm: %s'
Arguments: ('a24ab403c93811eb96ee5cc9d3a28a56',)
```

## Versions

* Spyder version: 5.0.0 
* Python version: 3.9.5
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  1.6.0 (OK)
cookiecutter >=1.6.0          :  1.7.2 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0               :  7.22.0 (OK)
jedi =0.17.2                  :  0.17.2 (OK)
jsonschema >=3.2.0            :  3.2.0 (OK)
keyring >=17.0.0              :  23.0.1 (OK)
nbconvert >=4.0               :  6.0.7 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
paramiko >=2.4.0              :  2.7.2 (OK)
parso =0.7.0                  :  0.7.0 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.9.0 (OK)
pylint >=1.0                  :  2.7.4 (OK)
pyls >=0.36.2;<1.0.0          :  0.36.2 (OK)
pyls_black >=0.4.6            :  0.4.6 (OK)
pyls_spyder >=0.3.2           :  0.3.2 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=0.5.7             :  1.0.2 (OK)
qtconsole >=5.0.3             :  5.0.3 (OK)
qtpy >=1.5.0                  :  1.9.0 (OK)
rtree >=0.8.3                 :  0.9.7 (OK)
setuptools >=39.0.0           :  52.0.0.post20210125 (OK)
sphinx >=0.6.6                :  4.0.1 (OK)
spyder_kernels >=2.0.1;<2.1.0 :  2.0.1 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3;<2.0.0      :  1.0.2 (OK)
zmq >=17                      :  20.0.0 (OK)

# Optional:
cython >=0.21                 :  None (NOK)
matplotlib >=2.0.0            :  None (NOK)
numpy >=1.7                   :  1.20.3 (OK)
pandas >=1.1.1                :  None (NOK)
scipy >=0.17.0                :  1.6.3 (OK)
sympy >=0.7.3                 :  None (NOK)
```

**Comment by dalthviz** [comment]
Hi @trierweiler thanks for the feedback! A fix for this issue will be part of Spyder 5.0.4: https://github.com/spyder-ide/spyder/pull/15777. Also, even after the release, probably you will need to use the `conda-forge` channel to get the update (currently the default channel doesn't have the most recent releases). Taking this into account, maybe creating a new env using the `conda-forge` channel instead of the default anaconda channel could help, let us know :)

**Comment by dalthviz** [comment]
Closing due to lack of response
